packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -88,6 +88,10 @@ import {
   PullRequestsStatsCard,
   Router as PullRequestsRouter,
 } from '@roadiehq/backstage-plugin-github-pull-requests';
+import {
+  isPluginApplicableToEntity as isSplunkOnCallAvailable,
+  SplunkOnCallCard,
+} from '@backstage/plugin-splunk-on-call';
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?